GUI/SearchTaskWindow.xaml.cs

```diff
@@ -183,7 +183,9 @@ private void UpdateFieldsFromTask(SearchTask task)
             useProvidedPrecursor.IsChecked = task.CommonParameters.UseProvidedPrecursorInfo;
             maxDegreesOfParallelism.Text = task.CommonParameters.MaxDegreeOfParallelism.ToString();
             disposeOfFilesWhenDone.IsChecked = task.SearchParameters.DisposeOfFileWhenDone;
-
+            allAmbiguity.IsChecked = task.CommonParameters.ReportAllAmbiguity;
+            someAmbiguity.IsChecked = !task.CommonParameters.ReportAllAmbiguity;
```

Comment:
Is this needed?

Reply:
Yes, otherwise if "someAmbiguity" was checked and then the toml is reopened, neither radiobutton will be checked. "someAmbiguity" radiobutton is not really necessary for the code to function, but assists the user in letting them know what happens if "allAmbiguity" radiobutton is unchecked.

Reply:
Maybe just have one check box? Ambiguity!